### ISSUE TITLE
يَعْرِفون أَنْطَلِق

### ISSUE THREAD

**Issue by htot** [cvzi/Arabic-Transcription#4]
This
يَعْرِفون
أَنْطَلِق
translates to "ya3rifuu2anTaliq" but should be (I think)
ya3rifuun
2anTaliq

T.b.h. I only tried this on the "live" version.

**Comment by cvzi** [comment]
Yes you're right. There seems to be a bug with the `أَ`. It deletes the previous character(s). If I put two spaces between the words, it seems to do it correctly.

I don't use this anymore myself and I haven't updated it in years, so I probably won't try to fix this, sorry 😐
